src/software/SfM/main_ComputeFeatures.cpp

```diff
@@ -257,11 +258,20 @@ int main(int argc, char **argv)
     << "[-m|--describerMethod]\n"
     << "  (method to use to describe an image):\n"
     << "   SIFT (default),\n"
+    << "   SIFT_FLOAT to use SIFT stored as float,\n"
     << "   AKAZE_FLOAT: AKAZE with floating point descriptors,\n"
     << "   AKAZE_MLDB:  AKAZE with binary descriptors\n"
+#if HAVE_CCTAG
```

Comment:
not here, but few lines above, I think we lost the doc for `"[-o|--outdir path] \n"`